Tests/FunctionalTests/MiscellaneousTests.swift

```diff
@@ -686,7 +686,7 @@ final class MiscellaneousTestCase: XCTestCase {
                 guard case SwiftPMError.executionFailure(_, _, let stderr) = error else {
                     return XCTFail("invalid error \(error)")
                 }
-                XCTAssert(stderr.contains("error: You don’t have permission"), "expected permissions error. stderr: '\(stderr)'")
+                XCTAssert(stderr.contains("error: You don’t have permission") || stderr.contains("invalid access"), "expected permissions error. stderr: '\(stderr)'")
```

Comment:
issue (possibly-blocking): I this is also update a test case.  do we know if this new expectation is expected, or whether it's a regression?